packages/make-styles/src/makeStyles.test.ts

```diff
@@ -5,13 +5,6 @@ import { MakeStylesRenderer } from './types';
 
 expect.addSnapshotSerializer(makeStylesRendererSerializer);
 
-function createFakeDocument(): Document {
```

Comment:
no need for this as new JSDOM has proper DOM structure

Reply:
if you're removing fake doucments please add a cleanup phase like in [9b3a49b](https://github.com/microsoft/fluentui/pull/20827/commits/9b3a49ba42ae2e0baa5bdf2869186d121353efc3)

since the renderer will keep appending style tags between each test

Reply:
you mean to add following ? 

```
  beforeEach(() => {
    document.head.innerHTML = '';
  });
```

Reply:
https://github.com/microsoft/fluentui/pull/20716/commits/3aebc6b649708e5a82320ee6d1a70e4ac3cac4dc